src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -127,11 +127,12 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
       // Check if not XPRO and BC restorable
       if (!this.isSelectedXproAndRestorable && this.isBcRestorable) {
         this.setLocation(Location.BC)
-        const corpType = this.getSearchBusiness.legalType as unknown as CorpTypeCd
+        const corpType = this.getSearchBusiness?.legalType as unknown as CorpTypeCd
```

Comment:
Because conditional chaining may return undefined, and an variable with a value of undefined is not reactive, provide a fallback value of null. (I think corpTypeToEntityType handles it but it's better to not depend on that.)